samcli/lib/build/workflow_config.py

```diff
@@ -0,0 +1,67 @@
+"""
+Contains Builder Workflow Configs for different Runtimes
+"""
+
+from collections import namedtuple
+
+
+CONFIG = namedtuple('Capability', ["language", "dependency_manager", "application_framework", "manifest_name"])
+
+GO_MOD_CONFIG = CONFIG(
+            language="go",
+            dependency_manager="modules",
+            application_framework=None,
+            manifest_name="go.mod")
+
+PYTHON_PIP_CONFIG = CONFIG(
+                language="python",
+                dependency_manager="pip",
+                application_framework=None,
+                manifest_name="requirements.txt")
+
+NODEJS_NPM_CONFIG = CONFIG(
+                language="nodejs",
+                dependency_manager="npm",
+                application_framework=None,
+                manifest_name="package.json")
+
+RUBY_BUNDLER_CONFIG = CONFIG(
+                language="ruby",
+                dependency_manager="bundler",
+                application_framework=None,
+                manifest_name="Gemfile")
+
+
+class UnsupportedRuntimeException(Exception):
+    pass
+
+
+def get_workflow_config(runtime):
+    """
+    Get a workflow config that corresponds to the runtime provided
+
+    Parameters
+    ----------
+    runtime str
+        The runtime of the config
+
+    Returns
+    -------
+    namedtuple(Capability)
+        namedtuple that represents the Builder Workflow Config
+    """
+
+    workflow_config_by_runtime = {
+        "go1.x": GO_MOD_CONFIG,
+        "python2.7": PYTHON_PIP_CONFIG,
+        "python3.6": PYTHON_PIP_CONFIG,
+        "python3.7": PYTHON_PIP_CONFIG,
+        "nodejs6.10": NODEJS_NPM_CONFIG,
```

Comment:
node4.x?

Reply:
[4.x is depreciated by Lambda](https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html). I can add it back but we need to figure out how to depreciate these when Lambda no longer supports them.